cli/pkg/cmd/create/create.go

```diff
@@ -49,6 +53,20 @@ func create(cmd *cobra.Command, args []string) {
 	w := console.GetMiddleware(func(input string) string {
 		return color.Colorize(color.Cyan, "#\t"+input)
 	})
+
+	client := analytics.New("7OOYJk8lQ2jfmZXKz5jJuvcK50BNDHit")
+	var rtm runtime.MemStats
+	runtime.ReadMemStats(&rtm)
+	track(client, analytics.Identify{
+		Traits: analytics.NewTraits().
+			Set("provider", providerName).
+			Set("numCpu", runtime.NumCPU()),
```

Comment:
This is about the CPU on the machine that runs the CLI right? So it is only meaningful for Minikube.